README - INSTALL INSTRUCTIONS/QBCore/README.md

```diff
@@ -261,3 +309,4 @@ Delete `qb-apartments/config.lua` references in `qb-garages` fxmanifest.lua.
 2. [fivem-freecam](https://github.com/Deltanic/fivem-freecam)
 3. [ox_lib](https://github.com/overextended/ox_lib/releases)
 4. [ox_target](https://github.com/overextended/ox_target) or [qb-target](https://github.com/qbcore-framework/qb-target)
+5. [ox_doorlock](https://github.com/overextended/ox_doorlock) or [ox_doorlock](https://github.com/qbcore-framework/qb-doorlock)
```

Comment:
We could consider adding the latest version which it works with with something like (tested and works with version X). Then "devs" won't be able to complain.